contracts/StrategyRegistry.vy

```diff
@@ -0,0 +1,61 @@
+# @version 0.2.12
+
+governance: public(address)
+pendingGovernance: public(address)
+
+event NewGovernance:
+    governance: address
+
+interface Strategy:
+    def name() -> String[64]: view
+    def apiVersion() -> String[8]: view
+
+struct StrategyInfo:
+  name: String[64]
+  apiVersion: String[8]
+
+strategyVersions: public(HashMap[StrategyInfo, address])
+
+@external
+def __init__():
+    self.governance = msg.sender
+
+@external
+def setGovernance(governance: address):
+    """
+    @notice Starts the 1st phase of the governance transfer.
+    @dev Throws if the caller is not current governance.
+    @param governance The next governance address
+    """
+    assert msg.sender == self.governance  # dev: unauthorized
+    self.pendingGovernance = governance
```

Comment:
I would emit an event when the `pendingGovernance` variable is changed, so we can track it offchain with thegraph.

Reply:
So we have `log NewGovernance(msg.sender)`, what we have been doing with vaults is only emit when governance is accepted. Do you think `pendingGovernance` would also be a good thing to have  on the vault?

It seems to me it is.

Reply:
Yes, I think it would be good to have.

Reply:
https://github.com/yearn/yearn-vaults/pull/412